minimizer/minimize.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash

set -euo pipefail
```
http://redsymbol.net/articles/unofficial-bash-strict-mode/